deps/readies/bin/python2

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ ! -z $(command -v python2) ]; then
+	python2 "$@"
+else
+	python "$@"
```

Comment:
This doesn't ensure that you're running python2. The `else` clause here could well start `python3` if `python` symlinks to it (as it is customary in several setups)

Reply:
Actually, any venv will offer its own version of python as "python". So it might be indeed better to check for python2.7 before falling back to "python", and then checking the python version and bailing out if it's not 2... It's gets really comical at this stage. :-)
I actually don't use this script for anything practical, and it's initial purpose was to deal with systems that for some reason offer "python" as python2 while not offering python2 at all (I think some centos-es do that).